.github/workflows/ci.yml

```diff
@@ -166,3 +166,24 @@ jobs:
     - name: make install
       run: make install
       working-directory: ./build
+
+  external-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - name: package installs
+      run: |
+        sudo apt-get update
+        sudo apt-get -yq install bison dejagnu gettext keyutils ldap-utils libldap2-dev libkeyutils-dev python3 python3-paste python3-pyrad slapd tcl-dev tcl-thread tcsh python3-virtualenv virtualenv
+    - name: install cpanm and Test2::V0
```

Comment:
I suggest adding "(for gost-engine testing)"

Reply:
Done